src/types/types.ts

```diff
@@ -0,0 +1,28 @@
+export interface HttpResponse {
+    statusCode: number;
+    headers: {
+      "Access-Control-Allow-Origin": string;
+    }
+    body: string;
+  };
+
+  export enum StatusCode {
```

Comment:
I would propose to split this file to 2 separate files:
http.ts
product.ts

Reply:
Done